front/gatsby/src/components/Articles.module.scss

```diff
@@ -140,51 +178,26 @@
   margin-right: 0.5rem;
 }
 
+.filterByTags {
+  display: flex;
+  flex-wrap: wrap;
+  > li:not(:last-child) {
+    margin-right: 1rem;
+    margin-bottom: 0.5rem;
+  }
+  margin-top: 0.25rem;
+  margin-bottom: 0.5rem;
+}
```

Comment:
```suggestion
.filterByTags {
  display: flex;
  flex-wrap: wrap;
  gap: 0.5rem;
  margin-top: 0.5rem;
  margin-bottom: 1rem;
}
```

Reply:
D'après MDN `gap` n'est pas supporté par Safari :sob: 

https://developer.mozilla.org/fr/docs/Web/CSS/gap#compatibilit%C3%A9_des_navigateurs

Après j'avoue que les marges avec `:not(:last-child)` c'est un peu moche... peut être qu'il faudrait basculer sur un grid layout ?

Reply:
Ah zut, j'avais zappé :-(

Alors ma proposition tient pas.